### PR TITLE
Update CLI guide content

### PR DESCRIPTION
**Pull request by agraebe**
Replacing #927

### CODE REVIEW

File: src/pages/understand-stacks/command-line-interface.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+- **Fee Rate**: The transaction fee rate for this transaction. You can safely set a fee rate of 1 for Testnet
+- **Nonce**: The nonce is a number that needs to be incremented monotonically for each transaction from the account. This ensures transactions are not duplicated
+- **Private Key**: This is the private key corresponding to your account that was generated when
+
+The CLI command to use with these parameters is `send_tokens`:
+
+```bash
+$ stx send_tokens ST2KMMVJAB00W5Z6XWTFPH6B13JE9RJ2DCSHYX0S7 1000 200 0 381314da39a45f43f45ffd33b5d8767d1a38db0da71fea50ed9508e048765cf301 -t
+
+d32de0d66b4a07e0d7eeca320c37a10111c8c703315e79e17df76de6950c622c
```
Comment (yknl) [inline src/pages/understand-stacks/command-line-interface.md:109]: We now respond with the txid and an explorer link

Comment (agraebe) [reply src/pages/understand-stacks/command-line-interface.md:109]: good point, didn't look closely enough at this content

Comment (agraebe) [reply src/pages/understand-stacks/command-line-interface.md:109]: I realize the references arent' updated either: https://docs.blockstack.org/references/blockstack-cli

**Review by yknl — APPROVED** [review]

**Comment by agraebe** [comment]
@yknl is this ready to be merged?

**Comment by yknl** [comment]
Yes, not sure why the vercel deployment failed though.

**Comment by agraebe** [comment]
@aulneau could you take a look. it shows this as an error:

`TypeError: Cannot set property 'customBlocks' of undefined`

File: src/pages/understand-stacks/command-line-interface.md (hunk 1 above)
Comment (agraebe) [reply src/pages/understand-stacks/command-line-interface.md]: ping

**Pull request merged**